### ISSUE TITLE
Logarithmic spectrograph view bug

### ISSUE THREAD

**Issue by Ghostbird** [essej/freqtweak#4]
![image](https://user-images.githubusercontent.com/1202149/79973816-bdeb7e80-8487-11ea-9da7-aec8595804fb.png)

As you can see here, the spectrograph view actually only updates the left part of the view. Most of the view is just the last data from the moment I switched to logarithmic view.
It is as if the binning for the view is written to the buffer in the wrong way.

**Comment by wildi** [comment]
Hello 

I have a similar problem. In plot spectrogram I see basically only two lines. What I consider the next line is at the top of the plot one pixel in height (see enlarged image), the rest of the plot has no vertical variation as it can be seen in the image found in this repo.

I did a git clone on Ubuntu 20.04 (see installed package versions below).
If I compile and link on Raspberry Pi 4 I see the normal behavior.

Kind regards, bye wildi

fftw-dev/focal,now 2.1.5-4.2build2 amd64 [installed]
fftw2/focal,now 2.1.5-4.2build2 amd64 [installed]
libfftw3-bin/focal,now 3.3.8-2ubuntu1 amd64 [installed,automatic]
libfftw3-dev/focal,now 3.3.8-2ubuntu1 amd64 [installed]
libfftw3-double3/focal,now 3.3.8-2ubuntu1 amd64 [installed]
libfftw3-long3/focal,now 3.3.8-2ubuntu1 amd64 [installed,automatic]
libfftw3-quad3/focal,now 3.3.8-2ubuntu1 amd64 [installed,automatic]
libfftw3-single3/focal,now 3.3.8-2ubuntu1 amd64 [installed]

libwxbase3.0-0v5/focal,now 3.0.4+dfsg-15build1 amd64 [installed,automatic]
libwxbase3.0-dev/focal,now 3.0.4+dfsg-15build1 amd64 [installed,automatic]
libwxgtk3.0-gtk3-0v5/focal,now 3.0.4+dfsg-15build1 amd64 [installed,automatic]
libwxgtk3.0-gtk3-dev/focal,now 3.0.4+dfsg-15build1 amd64 [installed]
wx-common/focal,now 3.0.4+dfsg-15build1 amd64 [installed]
wx3.0-headers/focal,focal,now 3.0.4+dfsg-15build1 all [installed,automatic]

libsigc++-2.0-0v5/focal,now 2.10.2-1build1 amd64 [installed,automatic]
libsigc++-2.0-dev/focal,now 2.10.2-1build1 amd64 [installed]

jackd2-firewire/focal,now 1.9.12~dfsg-2ubuntu2 amd64 [installed,automatic]
jackd2/focal,now 1.9.12~dfsg-2ubuntu2 amd64 [installed]
jackd/focal,focal,now 5+nmu1 all [installed,automatic]
libjack-jackd2-0/focal,now 1.9.12~dfsg-2ubuntu2 amd64 [installed,automatic]
libjack-jackd2-dev/focal,now 1.9.12~dfsg-2ubuntu2 amd64 [installed]

freqtweak documenation:
![image](https://user-images.githubusercontent.com/9784134/90973013-65ec8380-e51e-11ea-9b09-b54dd7bc4b1b.png)


![image](https://user-images.githubusercontent.com/9784134/90972885-0b9ef300-e51d-11ea-9550-fe6527978441.png)

![image](https://user-images.githubusercontent.com/9784134/90972913-5de01400-e51d-11ea-9db2-bd3d1b8a54f4.png)

**Comment by wildi** [comment]
I forgot to mention that changed all occurrences of
-       _updateMS = 10; // start small
+       _updateMS = 1000; // start small

in order to make the effect, seen in detail screenshot, visible.

**Comment by acidtonic** [comment]
Was this ever fixed or worked on? Same issue happening with Ubuntu 20.04 consistently to the point the app is unusable.

**Comment by Ghostbird** [comment]
I don't think it was fixed, and as you say, it's unusable. I don't use this any more.
